user.py

```diff
@@ -1,9 +1,29 @@
+from cryptography.fernet import Fernet
+
+
 class User:
     def __init__(self, user_id: str, pin: str, balance: int):
         self.user_id = user_id
         self.pin = pin
         self.balance = balance
 
+    def _get_key(self) -> bytes:
```

Comment:
I think that here should be this underscore line, because we call this method to get the key and nowhere else